cmds/core/init/init_linux.go

```diff
@@ -2,18 +2,25 @@
 // Use of this source code is governed by a BSD-style
 // license that can be found in the LICENSE file.
 
+// +build !plan9
```

Comment:
This shouldn't be necessary given the filename?

Reply:
done